tests/writer_test.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""Tests for the artifact definitions readers."""
+
+import io
+import os
+import sys
+import unittest
+import yaml
+import json
+
+from artifacts import errors
+from artifacts import reader
+from artifacts import writer
+
+
+class ArtifactAsDictTest(unittest.TestCase):
```

Comment:
TestCase is a collection of tests. at the moment these tests seem out of place. I opt to move them to the reader tests.